doc/copp/Copp_Manager_Redesign_test_plan.md

```diff
@@ -80,9 +80,11 @@ The test will be supported on ptf32, ptf64, t1 and t2.
 ## Test cases
 ### Test cases #1 -  Verify 4 traps'default configuration
 1. Verify the 4 traps(arp, lacp, udld, ip2me) are installed  by sending traffic with the default config
+   (For traffic validation, please see the Notes)
 
 ### Test cases #2 - Add a new trap
 #### Pre-condition: The tested trap should be uninstalled on dut.
+#### Remove ip2me because bgp traffic can fall back to ip2me trap then interfere following traffic tests
```

Comment:
BRCM ASIC based platform has an invisible default trap which will cause traffic been trapped even we remove ip2me. I suggest you update the statement and just mention the ip2me as an example.

They commit to update test case to adapt them ASIC behavior.

Reply:
Change as below. Is it ok?
Since different vendor's ASIC might have some different invisible default traps will cause traffic been trapped, we alsoe need to remove them.
For example: Remove ip2me because bgp traffic can fall back to ip2me trap then interfere following traffic tests

Reply:
Change as below. Is it ok?
Since different vendor's ASIC might have some different invisible default traps will cause traffic been trapped, we alsoe need to remove them. For example:  Remove ip2me because bgp traffic can fall back to ip2me trap then interfere following traffic tests

Reply:
@Blueve 
Fixed.
Can you review it again?
Thanks